tests/middleware_exceptions/tests.py

```diff
@@ -162,11 +162,23 @@ def assert_exceptions_handled(self, url, errors, extra_error=None):
             self.assertEqual(value.args, (error, ))
 
     def assert_middleware_usage(self, middleware, request, view, template_response, response, exception):
-        self.assertEqual(middleware.process_request_called, request)
-        self.assertEqual(middleware.process_view_called, view)
-        self.assertEqual(middleware.process_template_response_called, template_response)
-        self.assertEqual(middleware.process_response_called, response)
-        self.assertEqual(middleware.process_exception_called, exception)
+        self.assertEqual(
+            (
+                middleware.__class__.__name__,
```

Comment:
yeah, this assertion is called multiple times within the same test for different middleware, so including the class name helps a lot with seeing where it's failing. And asserting on all the values together in a tuple, rather than one at a time, helps a lot too. I'll add a comment and commit this separately, then rebase.